workflows/execute_workflows.sh

```diff
@@ -92,65 +87,72 @@ do
 done
 
 # download the necessary models if not available
-
 echo "Download the necessary models if not available"
 if [[ ! -f $ROOT/models/ocrd-tesserocr-recognize/Fraktur_GT4HistOCR.traineddata ]]
 then
     mkdir -p "$ROOT"/models
     docker run --volume "$ROOT"/models:/usr/local/share/ocrd-resources -- ocrd/all:maximum ocrd resmgr download ocrd-tesserocr-recognize Fraktur_GT4HistOCR.traineddata
 fi
+if [[ ! -d $ROOT/models/ocrd-calamari-recognize/qurator-gt4histocr-1.0 ]]
+then
+    mkdir -p "$ROOT"/models
+    docker run --volume "$ROOT"/models:/usr/local/share/ocrd-resources -- ocrd/all:maximum ocrd resmgr download ocrd-calamari-recognize qurator-gt4histocr-1.0
+fi
 
 # execute this workflow on the existing data (incl. evaluation)
 
 cd "$WORKSPACE_DIR" || exit
 
+# create workspace for all OCR workflows.
+# each workflow has a separate workspace to work with.
+mkdir -p "$WORKSPACE_DIR"/tmp
+for WORKFLOW in "$OCRD_WORKFLOW_DIR"/*ocr.txt.nf
+do
+    WF_NAME=$(echo "$WORKFLOW" | cut -d'.' -f 1 | rev | cut -d'/' -f 1 | rev)
+    echo "Restore OCR-D workspaces from BagIts …"
+    for BAGIT in "$ROOT"/submodules/quiver-data/*.zip
+    do
+        ocrd zip spill "$BAGIT" -d "$WORKSPACE_DIR"/tmp > "$WORKSPACE_DIR"/log.log
+        # this is a workaround for a werid behaviour where 16_ant_complex.ocrd.zip is sometimes not
+        # correctly extracted (unknown reason)
```

Comment:
Strange - any error messages?

Reply:
Weird indeed. Can you open an issue in core so we can debug this?

Reply:
I discovered that this is caused somehow while processing the BagIts. I restructured the bash script (i.e. tried to make it less complex) and that solved the issue.

This is somewhat unsatisfactory, but I will not investigate this further unless the problem re-occurs.